packages/cli/src/commands/migrate/init-command.ts

```diff
@@ -0,0 +1,49 @@
+import { Command } from 'commander';
+import { Listr } from 'listr2';
+import { createLogger, format, transports } from 'winston';
+
+import {
+  initTask,
+  depInstallTask,
+  tsConfigTask,
+  lintConfigTask,
+  createScriptsTask,
+  validateTask,
+} from './tasks/index.js';
+import type { MigrateCommandOptions } from '../../types.js';
+
+// subcommand migrate init
+export const initCommand = new Command();
+
+initCommand
+  .name('init')
+  .description('Install required dependencies, setup tsconfig.json and eslint config for migration')
+  .action(async (options: MigrateCommandOptions) => {
+    await initCommandHandler(options);
+  });
+
+export async function initCommandHandler(options: MigrateCommandOptions): Promise<void> {
+  const loggerLevel = options.verbose ? 'debug' : 'info';
+  const logger = createLogger({
+    transports: [new transports.Console({ format: format.cli(), level: loggerLevel })],
+  });
+  const defaultListrOption = {
+    concurrent: false,
+    exitOnError: true,
+  };
+
+  const tasks = [
```

Comment:
the functionality we are looking for would make this child command atomic. we can combine the tasks in the migrate/index.ts command. for this command it shouldn't run any sibling tasks. 

`rehearsal migrate` <<< will run the tasks including init
`rehearsal migrate init` <<< will only run init